echo.go

```diff
@@ -506,7 +506,7 @@ func (e *Echo) URL(h HandlerFunc, params ...interface{}) string {
 
 // Reverse generates an URL from route name and provided parameters.
 func (e *Echo) Reverse(name string, params ...interface{}) string {
-	uri := new(bytes.Buffer)
+	uri := bytes.NewBuffer(nil)
```

Comment:
For consistency, please search and replace similar statements across the project.

Reply:
ok

Reply:
I think `var uri bytes.Buffer` is more reasonable.

Reply:
`var uri bytes.Buffer` not equal `uri := bytes.NewBuffer(nil)`

Reply:
fixed